src/quo2/components/gradient/gradient_cover/view.cljs

```diff
@@ -0,0 +1,18 @@
+(ns quo2.components.gradient.gradient-cover.view
+  (:require [quo2.components.gradient.gradient-cover.style :as style]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo.theme]
+            [react-native.linear-gradient :as linear-gradient]))
+
+(defn- view-internal
+  [{:keys [color] :or {color :blue}}]
+  (let [color-top    (colors/custom-color-by-theme color 50 50 20 20)
```

Comment:
2 things, we use `customization-color` as the name for this prop everywhere else.
 is this theme really necessary here?, looks like it's the same in light and dark.
 it can be  
 ```
 (let [color-top    (colors/custom-color customization-color 50 20)
        color-bottom (colors/custom-color  customization 50 0)]
   ```

Reply:
Yep, It's the same color in both modes. I will update it. 👍  
Thanks @J-Son89!